doc/modules/cassandra/pages/developing/cql/ddl.adoc

```diff
@@ -630,6 +630,7 @@ If the `enabled` option is set to `false`, no other options must be specified.
 
 | `chunk_length_in_kb` | 64 | On disk SSTables are compressed by block (to allow random reads).
 This option defines the size (in KB) of said block. See xref:cassandra:developing/cql/ddl.adoc#chunk_note[note] for further information.
+| `chunk_length` | 64KiB| alias to `chunk_length_in_kb` where size is specified with unit, available from CASSANDRA-12973.
```

Comment:
I don't think it's a good idea to add an alias for a configuration parameter, it would lead us eventually to a messy configuration and backwards compatibility maintenance problems. 

We already did some housekeeping here:
https://issues.apache.org/jira/browse/CASSANDRA-18742

I would say we have a few options:
- don't add `chunk_length` at all and use only `chunk_length_in_kb`;
- create another issue and deprecate the `chunk_length_in_kb` in favor of `chunk_length`

The issue we are working on is - "Move defaults to yaml" we shouldn't mix it with refactoring/deprecation:
https://issues.apache.org/jira/browse/CASSANDRA-12937

Reply:
@Mmuzaf ok I removed chunk_length, I do not have any strong feelings about that. If we want to introduce it and deprecate chunk_length_in_kb, we might just dedicate a separate ticket for that and do it as a standalone change as you suggested.